### DISCUSSION TITLE
Rendering JSON properly

### DISCUSSION THREAD

**Discussion by Hiba-Ed** [adonisjs/core#1730] [Help]
Here is my Product model 

```
'use strict'

/** @type {typeof import('@adonisjs/lucid/src/Lucid/Model')} */
const Model = use('Model')

class Product extends Model {
  static get createdAtColumn() { 
    return false
  }
  static get updatedAtColumn() { 
    return false
  }
}

module.exports = Product

```
When I do await Product.all(), I get an untidy looking array like so:
**[{"id":1,"name":"Apple Mac Pro Stand","price":999,"created_datetime":"2020-10-09T07:43:59.000Z","updated_datetime":null},{"id":2,"name":"Apple Macbook Pro 2019","price":1999,"created_datetime":"2020-10-09T07:43:59.000Z","updated_datetime":null}]**

Is there a way to neatly render it?

**Comment by w33zy** [comment]
Have you tried setting `serializeAs = null` on the `createdAt` and `updatedAt` properties?

See: https://preview.adonisjs.com/guides/models/serializing-models#hiding-properties

**Reply by Hiba-Ed** [reply]
Can you please let me know how to do this in Adonis 4.1?

I have done this:
```
class Product extends Model {
  static get createdAtColumn() { 
    serializeAs = null
    return false
  }
  static get updatedAtColumn() { 
    serializeAs = null
    return false
  }
}
```
I get the error **ReferenceError: serializeAs is not defined**. Am I missing a package? If so, please specify. Thank you.
